tests/framework/BaseYiiTest.php

```diff
@@ -84,4 +86,16 @@ public function testCreateObjectCallable()
             return true;
         }));
     }
+
+    /**
+     * @covers yii\BaseYii::setLogger()
+     * @covers yii\BaseYii::getLogger()
+     */
+    public function testSetupLogger()
+    {
```

Comment:
@klimov-paul, that do you think about this assertion:
$this->assertInstanceOf('yii\log\Logger', BaseYii::getLogger());

for this part:
```php
// yii2/framework/BaseYii.php:363
} else {
    return self::$_logger = static::createObject('yii\log\Logger');
}
```